PER-39, Syncdata testcases

nihallakshmankonchady opened the pull request.

Commit:
PER-39, Syncdata testcases

Signed-off-by: Nihal LK <nihal.lk@technoforte.co.in>

Sanjayss04 commented:
Before start reviewing this PR. Please once have a cross check on the comments made to the mentioned PR - https://github.com/mosip/performance-test-scripts/pull/130. 

Please try to cover all those review comments in this PR also, once you have completed then will further review it.

nihallakshmankonchady commented:
Updated the SyncdataHelper and Test Script with some minor modifications.

- Updated the CSV file with correct values. The endpoints are running fine now.
- Added response code assertion.
- Corrected the naming convention of test data files which are used in the Test Script.

nihallakshmankonchady commented:
@Sanjayss04 .. I have added the description for PR
